### PR TITLE
[INTERNAL] JSDoc: Fix processors/nonAsciiEscaper.getEncodingFromAlias

### PR DESCRIPTION
**Pull request by matz3**

### CODE REVIEW

File: lib/processors/nonAsciiEscaper.js [hunk 1]
```diff
@@ -92,12 +92,16 @@ const encodingMap = {
  * specific encoding name such as "utf8" or "latin1". Simplifies usage of nonAsciiEscaper encoding
  * option such that it can be used standalone without the respective task (e.g. in Splitter, Bundler and related projects).
  *
+ * @public
+ * @alias module:@ui5/builder.processors.nonAsciiEscaper․getEncodingFromAlias
```
Comment (matz3) [inline lib/processors/nonAsciiEscaper.js:96]: This is a special character (not a normal dot), as otherwise the template does not render the function.

Comment (codeworrior) [reply lib/processors/nonAsciiEscaper.js:96]: LFTM

**Comment by matz3** [comment]
Rebased & commit message updated

**Review by codeworrior — APPROVED** [review]
LGTM

File: lib/processors/nonAsciiEscaper.js (hunk 1 above)
Comment (codeworrior) [reply lib/processors/nonAsciiEscaper.js:97]: Not sure what I had in mind with the 'F': funny? fancy? forbidden?

Comment (matz3) [reply lib/processors/nonAsciiEscaper.js:97]: "Living for the Moment" aka YOLO?

**Pull request merged**